tasker/templates/form_errors.html

```diff
@@ -1,9 +1,7 @@
 {% macro field_error(field) %}
     {% if field.errors %}
-        <ul>
             {% for error in field.errors %}
-            <li class="error-msg">{{ error }}</li>
+            <p class="error-msg">{{ error }}</p>
```

Comment:
Not sure about this one.  It doesn't seem like correct HTML to put a bunch of paragraph tags inside an unordered list tag.  Maybe we want to add a different kind of error macro. Or get rid of the unordered list altogether?

Reply:
I think I removed the ul

Reply:
You're correct Jeremy, we have to pick one. Either we have no list (remove ul and li tags) and just use paragraph tags, or we use only the list tags inside the unordered list. I think the paragraph with no list would be the better choice in this case just because it will require less styling to make it look right.

Reply:
That's what I have, a paragraph with no ul or li. Right?

Reply:
@ekmartens do I need to make any changes?